submissions/ustymchyk/js-post-oop/index.js

```diff
@@ -203,10 +241,9 @@ const puppy = ruby.bornCreature(asher, 'Sushi', 'Sid');
    print('ABC');
    print('<strong>ABC</strong>');
    print('<strong>ABC</strong>', 'div');
-
    print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
    print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
    print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
    */
 
-[...men, ...women, luna, leo, kitty, asher, ruby, puppy].forEach(creature => print(creature.introduce()));
+[...men, ...women, luna, leo, kitty, asher, ruby, puppy].forEach(creature => print(creature.toString()));
```

Comment:
Here is the bonus:
`.toString` is a magic method defined for every object in JS. It called implicitly in string context. It is `Object.toString` that is responsible for `[Object object]` output in console under node or via `alert` in browser.
`print` function ensures string context for its arguments, so `print(creature)` will work as expected.